### ISSUE TITLE
Credit Card Type in Contribution Export

### ISSUE THREAD

**Issue by cmmadmin** [iATSPayments/com.iatspayments.civicrm#243]
We need to be able to export the credit card type (Visa, Mastercard, etc) when we export contributions. This is important for our reconciliation process. Is this saved somewhere in the CiviCRM database by the iATS extension?

It seems like it may be stored in the database as I see some non-NULL values with:
SELECT * FROM `civicrm_financial_trxn` WHERE `card_type_id` IS NOT NULL

But, I do not see Card Type in the fields that are available for export when using "Export Contributions" (from the Actions menu on a Find Contributions results page).

CiviCRM 5.0.0, iATS 1.6.1

**Comment by adixon** [comment]
Yes, it's included in the iats-specific journal report. I can also share with you a customized version of the Contribution Detail report that adds in this data. Something like it will be included into the next release.

I'll note that credit card information (card type, expiry and mask) does now have some fields in core that sometimes get populated, but not reliably.

**Comment by cmmadmin** [comment]
Where can we find the iats-specific journal report?

And, yes would like to have the customized version of the Contribution Detail report. Can you send it to me?

Thank you!

**Comment by cmmadmin** [comment]
Hello Alan,

Can you help me find the iats-specific journal report that you mentioned? And would you mind sharing the customized version of the Contribution Detail report as well?
